### ISSUE TITLE
Enhancement Request: Allow missing covers to be fetched from other providers than Last.fm

### ISSUE THREAD

**Issue by chrober** [CDrummond/cantata#988]
In the "General" tab of Cantata's "Interface Settings" there is a checkbox "Fetch missing covers from Last.fm". It would be nice to have Cantata fetching covers from other providers as well, e.g. CovertArt Archive or iTunes. The drawback of Last.fm covers is they usually have a rather low resolution of 300x300 pixels.

**Comment by CDrummond** [comment]
Then use the cover dialog to look or better. Or, better still, provide covers before starting Cantata.
